docs/testing.md

```
@@ -1,3 +1,235 @@
+  Background:
+    Given "tags" terms:
+      | name              |
+      | [TEST] Topic 1    |
+      | [TEST] Topic 2    |
+
+    And the following media "image" exist:
+      | name                    |
+      | [TEST] Featured Image 1 |
+  ```
+
+3. Test all visible fields
```

markdownlint-cli2 (0.20.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)